### DISCUSSION TITLE
part-ii/2017-75/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#170] [Question comments]
Part II 2017 Graph Theory: Paper 3, Section II, H 

https://questions.tripos.org/part-ii/2017-75/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

The first part is bookwork. 

A red-blue colouring of the edges of $K_{2t-2}$ with no red $K_t$ or blue odd cycle is given by colouring a Hamiltonian cycle blue (this is an even cycle) and every other edge red (any choice of $t$ vertices has to contain $2$ that are adjacent in the Hamiltonian cycle by the pigeonhole principle).

For the last part, suppose we have a colouring of the edges of $K_{2t-1}$ with no blue odd cycles. Then the graph induced by the blue edges is bipartite, so one of its vertex classes must have size at least $t$. But all the edges within this vertex class are red, so we get a red $K_t$.

</details>
